packages/idyll-components/src/desmos.js

```diff
@@ -95,7 +95,7 @@ class Desmos extends React.Component {
     });
   }
 
-  componentWillUpdate(nextProps) {
+  UNSAFE_componentWillUpdate(nextProps) {
```

Comment:
I think this is safe to change to `componentDidUpdate`

Reply:
Cool will try all these and update the PR.